### DISCUSSION TITLE
SOLVED use {chat_id} as the service protocol number

### DISCUSSION THREAD

**Discussion by GuilhermeDeLucca** [LiveHelperChat/livehelperchat#1827] [General]
I would like to send a text to the visitor saying:

This is your protocol number {chat_id}

However, this variable is not displayed to the user, becoming {chat_id}

What am I doing wrong? This variable is just for the API, or I can do what I want directly in the BOT interactions

`[
    {
        "_id": "zOX4gqlSR",
        "type": "text",
        "content": {
            "text": "This is your protocol number {chat_id}"
        }
    }
]`

![Captura de Tela 2022-11-30 às 06 41 41](https://user-images.githubusercontent.com/9898902/204762162-3c2b55a9-ffba-4004-8f33-ab568f9e59ac.png)

**Comment by remdex** [comment]
Use `{args.chat.id}`
https://doc.livehelperchat.com/docs/bot/bot-text#replaceable-variables

**Reply by GuilhermeDeLucca** [reply]
Perfect worked correctly!

However, I still don't understand how it works {args.

https://doc.livehelperchat.com/docs/bot/bot-text#replaceable-variables
In the documentation we have:

Other:

{{ip}} - visitor ip. If you are running background worker this value will be localhost.
{{footprint}} - last 25 viewed pages url's
{{chat_id}} - chat ID
{{timestamp}} - unix timestamp

Thank you for your help!

I really hadn't thought of {args.<>} because it is not clear to me

**Reply by remdex** [reply]
You are mixing two things. Replace variables for the text messages etc. https://doc.livehelperchat.com/docs/bot/bot-text#replaceable-variables they have single brackets.

Rest API bodu has two brackets
https://doc.livehelperchat.com/docs/bot/rest-api

`args.` you can access any passed argument and it's sub attribute see chat definition
https://api.livehelperchat.com/#/

Or directly PHP class
https://github.com/LiveHelperChat/livehelperchat/blob/master/lhc_web/lib/models/lhchat/erlhcoreclassmodelchat.php
